app/view_models/workarea/admin/product_view_model.decorator

```diff
@@ -14,6 +14,16 @@ module Workarea
         end
     end
 
+    def variant_details
```

Comment:
Seems like this is the same as this method, can you reuse it? https://github.com/workarea-commerce/workarea/blob/master/admin/app/view_models/workarea/admin/product_view_model.rb#L96

Reply:
that method is really problematic, and looks like we don't use it. Probably should get rid of it. Or at least rename it like this one.